packages/at_policy/lib/src/policy/impl.dart

```diff
@@ -130,32 +117,25 @@ class PolicyServiceImpl with AtClientBindings implements PolicyService {
           '\n$st');
     }
 
-    PolicyResponse policyResponse;
     AtRpcResp rpcResponse;
     try {
-      policyResponse = await handler.getPolicyDetails(policyRequest);
-      Map<String, dynamic> responsePayload = policyResponse.toJson();
-      if (responseTransformer != null) {
-        responsePayload = await responseTransformer!(responsePayload);
-      }
+      PolicyResponse policyResponse =
+          await handler.getPolicyDetails(policyRequest);
       rpcResponse = AtRpcResp(
           reqId: rpcRequest.reqId,
           respType: AtRpcRespType.success,
-          payload: responsePayload);
+          payload: policyResponse.toJson());
     } catch (e) {
       logger.severe('Exception: $e');
-      policyResponse = PolicyResponse(
+      rpcResponse = AtRpcResp(
+        reqId: rpcRequest.reqId,
+        respType: AtRpcRespType.error,
+        payload: PolicyResponse(
+          message: 'Exception: $e',
+          policyDetails: [],
+        ).toJson(),
         message: 'Exception: $e',
```

Comment:
Is message supposed to be there twice? I see it was moved into the payload, but also kept in response.

Reply:
Yes supposed to be there twice, thanks Xavier